src/components/carousel/ThumbnailNavigationButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react';
+import { chakra, useMultiStyleConfig } from '@chakra-ui/react';
+
+import { Direction } from './NavigationButton';
+import { ChevronLeftSmallIcon, ChevronRightSmallIcon } from '../icons';
+import Flex from '../flex';
+
+interface Props {
+  onClick: () => void;
+  direction: Direction;
+}
+
+const icons = {
+  previous: <ChevronLeftSmallIcon />,
+  next: <ChevronRightSmallIcon />,
+};
+
+const ThumbnailNavigationButton: FC<Props> = ({ direction, onClick }) => {
+  const { paginationButtonContainer, paginationButton } = useMultiStyleConfig(
+    'Carousel',
+    {
+      variant: 'fullScreen',
+    }
+  );
+  const side = direction === 'previous' ? { left: '0' } : { right: '0' };
+
+  return (
+    <chakra.button
+      onClick={onClick}
+      {...side}
+      aria-label={`scroll to ${direction} thumbnail group`}
+      aria-controls="thumbnails-wrapper"
+      __css={paginationButtonContainer}
```

Comment:
Why did you call the styles for the button `paginationButtonContainer` and the ones for the flex `paginationButton`?

Reply:
good point. Let me rename them :D